types/edit-json-file/index.d.ts

```diff
@@ -22,7 +25,7 @@ declare namespace editJsonFile {
         /** Get value at path. */
         get(path?: string): any;
         /** Set value at path. */
-        set(path: string, value: any): JsonEditor;
+        set(path: string, value: any, options?: SetValueOptions): JsonEditor;
```

Comment:
I think `set.Options` is enough
```suggestion
        set(path: string, value: any, options?: set.Options): JsonEditor;
```

Reply:
Yup.

Reply:
Good catch. Thought that wasn't exported at first but turns out it is available for me to use. Updated.